WhateverGreen/kern_igfx.cpp

```diff
@@ -996,11 +996,55 @@ OSObject *IGFX::wrapCopyExistingServices(OSDictionary *matching, IOOptionBits in
 	return FunctionCast(wrapCopyExistingServices, callbackIGFX->orgCopyExistingServices)(matching, inState, options);
 }
 
+bool IGFX::removeVTSupportedProfile2(IOService *that) {
+	if (!that)
+		return false;
+
+	auto hevcDecodeCap = OSDynamicCast(OSDictionary, that->getProperty("IOGVAHEVCDecodeCapabilities"));
+	if (!hevcDecodeCap)
+		return false;
+
+	auto hevcDecodeCapCpy = OSDictionary::withDictionary(hevcDecodeCap);
+	if (!hevcDecodeCapCpy)
+		return false;
+
+	auto vtSuppProf = OSDynamicCast(OSArray, hevcDecodeCapCpy->getObject("VTSupportedProfileArray"));
+	if (!vtSuppProf) {
+		hevcDecodeCapCpy->release();
+		return false;
+	}
+
+	auto vtSuppProfCpy = OSArray::withArray(vtSuppProf);
+	if (!vtSuppProfCpy) {
+		hevcDecodeCapCpy->release();
+		return false;
+	}
+
+	auto count = vtSuppProfCpy->getCount();
+	bool found = false;
+	for (auto i = 0; i < count; i++) {
```

Comment:
auto here will default to int, better use size_t for both count and i.

Reply:
`getCount()` returns `unsigned int`, so I rather followed it: https://github.com/acidanthera/WhateverGreen/commit/e6f90ecbcc605c608a2d945793ac7ab706579179